tests/GoTest.sh

```diff
@@ -20,26 +20,20 @@ go_path=${test_dir}/go_gen
 go_src=${go_path}/src
 
 # Emit Go code for the example schemas in the test dir:
-../flatc -g --gen-object-api -I include_test monster_test.fbs optional_scalars.fbs
+../flatc -g --gen-object-api -I include_test -o ${go_src} monster_test.fbs optional_scalars.fbs
+../flatc -g --gen-object-api -I include_test/sub -o ${go_src} include_test/foo.fbs
+../flatc -g --gen-object-api -o ${go_src} include_test/sub/header.fbs
 
 # Go requires a particular layout of files in order to link multiple packages.
 # Copy flatbuffer Go files to their own package directories to compile the
 # test binary:
-mkdir -p ${go_src}/MyGame/Example
-mkdir -p ${go_src}/MyGame/Example2
 mkdir -p ${go_src}/github.com/google/flatbuffers/go
 mkdir -p ${go_src}/flatbuffers_test
-mkdir -p ${go_src}/optional_scalars
+mkdir -p ${go_src}/Pizza
 
-cp -a MyGame/*.go ./go_gen/src/MyGame/
```

Comment:
Can we minimize the changes to this script?

If we want to add a couple of new imports to the go tests I don't think we need to change this file that much.

Reply:
Changes are for cleaning purpose mostly. Btw, I will try to make it better